src/main/java/com/powsybl/openloadflow/network/LfContingency.java

```diff
@@ -38,12 +39,14 @@ public class LfContingency {
 
     private final Map<LfBus, PowerShift> busesLoadShift;
 
+    private final Set<String> originalPowerShiftIds;
```

Comment:
At first reading it was really not clear to me that this is load IDs.

Reply:
It is not only load ids but also the other equipments converted as power shift as LCC converter stations.